playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md

```diff
@@ -0,0 +1,24 @@
+### Description
+Enables users to add longer text to a form
+
+### When to Use
+For long answers from users
+
+### When not to Use
+For short answers from users, you can use a Text Input instead
+
+### How to use
+| <img width="600" alt="Screenshot 2023-06-14 at 1 50 02 PM" src="https://github.com/powerhome/playbook/assets/73671109/42084be1-84f1-4195-8257-a1cc09bec056"> | <img width="500" alt="2023-06-14 at 2 28 18 PM" src="https://github.com/powerhome/playbook/assets/73671109/8a53e409-5d93-4bc0-b3b2-a40f19588db5"> |
```

Comment:
I'm assuming you added an img tag so that you could tap into the `width`?

Reply:
Yes. Otherwise the images are way to big and look weird.

Reply:
I still think there is a better way to display the code examples and images so any advice is welcome!